src/Util/Requests.ts

```diff
@@ -5,11 +5,10 @@ import RequestTypes from "./RequestTypes";
 
 const FormData = require('form-data');
 
-const blobFromSync = (...args : any) => // @ts-ignore
-    import('node-fetch').then(({ blobFromSync }) => blobFromSync(...args));
+// TODO: needs different implementation
+const blobFromSync = (...args : any) => Promise.resolve({} as Blob)
```

Comment:
Is this going to work? There are clients that actually use blobFromSync to a break change is a no go. Is there a way of still coming and only important the blobFromSync if its currently node and not the browser?

And altnerative solution I was contemplating was writing a custom blobFromSync. The original blobFromSync function is:

https://github.com/node-fetch/fetch-blob/blob/main/from.js#L23

``` const blobFromSync = (path, type) => fromBlob(statSync(path), path, type)```

And from fromBlob is:
https://github.com/node-fetch/fetch-blob/blob/main/from.js#L46

```
const fromBlob = (stat, path, type = '') => new Blob([new BlobDataItem({
  path,
  size: stat.size,
  lastModified: stat.mtimeMs,
  start: 0
})], { type })
```

Looking at this package that has fs for the browser: https://www.npmjs.com/package/node-browserfs

It might be possible to write one for the browser. If nodejs import node-fetch, else import node-browserfs.

Is a version of that logic possible OR what why can be implemented not break for current clients?

Reply:
So if I understand correctly, that function creates blob from a provided file path like `/Users/Tomas/Desktop/video.mp4`. But you cannot do it inside the browser, there is no path. 
In the browser you need to get a `File` or a `Blob` directly.

Reply:
But yes, I will try it this way:
- if it is the browser and the variable is not a File, throw an error,
- if it is the NodeJS, then call the `blobFromSync` to create a Blob